Sources/SwiftDocCTestUtilities/FilesAndFolders.swift

```diff
@@ -93,51 +93,41 @@ public struct InfoPlist: File, DataRepresentable {
     /// The information that the Into.plist file contains.
     public let content: Content
 
-    public init(displayName: String, identifier: String, versionString: String = "1.0", developmentRegion: String = "en") {
+    public init(displayName: String, identifier: String? = nil, versionString: String = "1.0") {
```

Comment:
I am not familiar with this, but what was "developmentRegion" and why was it removed?

Reply:
It's a configuration that we never parsed and never did anything with so there wasn't any point in writing it to the Info.plist in all the tests.